packages/fluentui/react-northstar/test/specs/components/Menu/Menu-test.tsx

```diff
@@ -54,6 +54,37 @@ describe('Menu', () => {
       expect(items[0].onClick).toHaveBeenCalled();
     });
 
+    it('should open on hover and keep open on click', () => {
+      const items: MenuItemProps[] = getNestedItems();
+      items[1].on = 'hover';
+      const menuItems = mountWithProvider(<Menu items={items} />).find('MenuItem');
+
+      expect(
+        menuItems
+          .at(1)
+          .at(0)
+          .find('a')
+          .first()
+          .getDOMNode()
+          .getAttribute('aria-expanded'),
+      ).toBe('false');
+
+      menuItems
+        .at(1)
+        .simulate('mouseover')
```

Comment:
Please should we have a separate check after `mouseover` and after `click`? To see that menu actually opened on hover and kept opened after the click?